### ISSUE TITLE
Training on 4 GPU server

### ISSUE THREAD

**Issue by hareskoruz** [open-mmlab/mmaction2#1098]
I have run demo_spatiotemporal.py  with regular all classes/also custom classes. But my server is 4 GPU and the pretrained one is 8 GPU . the detection for custom classes is working, but not detecting accordingly, Is there any canges in config file or demo _spatiotemporal.py File for our GPU?

**Comment by irvingzhang0512** [comment]
Hi, what do you mean by `the detection for custom classes is working, but not detecting accordingly`? Could you please give us an example?

**Comment by hareskoruz** [comment]
@irvingzhang0512 
Hi, The detection is randomly showing different actions for different gpu, let's say i have 4 gpu .
and is there any changes needs to be done on learning rate as, they've run on 8 gpu server

**Comment by hareskoruz** [comment]
Hi @irvingzhang0512 , can u suggest how to change config file for 4 GPU server?

**Comment by irvingzhang0512** [comment]
Sorry about my poor english. I didn't catch you. 

There is no need to modify configs when number of gpus is changed. And I thought `demo_spatiotemporal.py` only worked for single gpu mode...

**Comment by hareskoruz** [comment]
But, the detection is not accurate for `slow only custom classes`

**Comment by irvingzhang0512** [comment]
I think spatio-temporal action detection is still in the research stage, and it is still far from industrial applications

**Comment by hareskoruz** [comment]
Can u suggest me which of your models will suite for real time action monitoring?

**Comment by hareskoruz** [comment]
Hi @irvingzhang0512 , Can you tell me how much space required for AVA dataset downloading and preparation?
